api/user/__init__.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint
+from .authors import authors_bp
+from .followers import followers_bp
+from .posts import posts_bp
+
+user_bp = Blueprint("users", __name__)
+user_bp.register_blueprint(authors_bp)
```

Comment:
Since the all the author related endpoints all start with /authors.
I think we can register all the author related endpoints using
`.register_blueprint(authors_bp, url_prefix="/authors"`

Reply:
I like this idea with prefixes...shouuld've used this _before_ I wrote all these out :)